### DISCUSSION TITLE
import does not complete...

### DISCUSSION THREAD

**Discussion by st-man-hori** [mozilla/hubs#3910] [Hubs Cloud Questions] [answered]
Hi

I tried import avatar in hubs admin panel.
But import does not complete.

As per the attached image.

Do you have any solutions?

【My environment】
AWS EC2 instance : t3.midium
SpecifyInboundCidr :  all allowed
Web browze :  fire fox   or  chrome

browze developer tools -> network error
```
ncaught (in promise) TypeError: Cannot read property '0' of undefined
    at r (import-content.js:220)
    at r.next (<anonymous>)
    at t.<anonymous> (import-content.js:202)
    at Generator.next (<anonymous>)
    at bY (admin-f16a01ee5570c6c28249.js:305)
    at o (admin-f16a01ee5570c6c28249.js:305)
POST https://dev.xrshophub.com/api/v1/avatars 403
```

regards.
  
![image](https://user-images.githubusercontent.com/77187521/107925709-24620000-6fb8-11eb-9212-3c585b9f06bd.png)

**Comment by robinkwilson** [comment]
Can you share the link to the avatar or `.pack` file, if it's public? 
A 403 error means "forbidden", can you double check that your account is an admin account?

**Comment by pattersonbl2** [comment, marked as answer]
Marking as answered, please unmark if you still need assistance
